src/Foundation.Find.Cms/Config/EnumSelectionFactory.cs

```diff
@@ -0,0 +1,43 @@
+﻿using EPiServer.Shell.ObjectEditing;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Foundation.Find.Cms.Config
```

Comment:
Change namespace to Foundation.Find.Cms.Facets.Config

Reply:
Done